apps/kyc-server/docs/api/kyc-reviews.md

```
@@ -0,0 +1,229 @@
+
+## Error Response Format
+All error responses follow this format:
+```json
+{
+  "error": "Error message describing what went wrong"
+}
+```
+
+## Examples
+
+### Create a KYC Review
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)